modules/create_timepoint/js/.gitignore

```diff
@@ -0,0 +1 @@
+*.js
```

Comment:
i think we're standardizing the .gitignore to be in the module directory with js/* in it

Reply:
reference https://github.com/aces/Loris/pull/4426